codeblocks.lua

```diff
@@ -42,6 +42,10 @@
 	end
 
 	function codeblocks.generateProject(prj)
+		--Kind "Utility" is not supported
+		if prj.kind == "Utility" then
```

Comment:
Should already be ignored by `valid_kinds = { "ConsoleApp", "WindowedApp", "StaticLib", "SharedLib" },`